### PR TITLE
v0.2.1: fix versions of giantswarm components 

### PR DESCRIPTION
**Pull request by kubasobon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.1] 2020-04-10
```
Comment (xh3b4sd) [inline CHANGELOG.md:10]: IMO we should make a separate PR for the release just before we create that tag. Until then all changes should be made separately.

File: service/service.go [hunk 2]
```diff
@@ -114,8 +114,7 @@ func New(config Config) (*Service, error) {
 
 		versionService, err = version.New(c)
 		if err != nil {
-			return nil, microerror.Maskf(err, "version.New")
-		}
+			return nil, microerror.Mask(err)		}
```
Comment (xh3b4sd) [inline service/service.go:117]: That needs `gofmt`.

**Comment by wtty-fool** [comment]
@xh3b4sd The build is failing and I can't figure out why. Could you take a look? Maybe it rings a bell for you.

**Comment by xh3b4sd** [comment]
I will have a look.

**Comment by xh3b4sd** [comment]
The build is fine now. 🙈

**Review by xh3b4sd — APPROVED** [review]
